### PR TITLE
Add more commonly used helper functions

### PR DESCRIPTION
**Pull request by ewilz**

### CODE REVIEW

**Comment by mikec** [comment]
looking good! 👍

**Comment by jpkcambridge** [comment]
Looks good to me.  Are these functions coming just from contracts-starter-project?  It looks like that has a slightly different set of helpers?

Also, kind of a nitpick, but the name "Add more redundant functions," while I get what it means here, sort of sounds to me like we're adding more redundancy.  Maybe a less ambiguous way to put it would be simply "adding more helper functions"?

**Comment by mikec** [comment]
@jpkcambridge agreed on the naming. I think these came over from the starter project? We'll also have to remove them from the starter project and add `lk-test-helpers` as a dependency there

**Comment by jpkcambridge** [comment]
@emaG3m Can you clarify on where these helpers were drawn from?  And any thoughts on the difference between what's here and what's in the starter project?

**Comment by ewilz** [comment]
Hey, sorry for the delay. I'm so used to getting email notifications for github comments. Yah, will definitely change the PR/commit name. 

I took the majority of these simply from the tests john and I have been writing in cointopia. I believe I originally got them from Pikcio, not sure where John was copying them from, perhaps just the other cointopia test files. Then I went to the starter-project to see if there were any other functions in there that I should add, and decided to pull in `isNonZeroAddress`. Are there any others from the starter project I should include? there's `getContract` and `getTruffleContract` which I did not add, and I'm not exactly sure their purpose. Perhaps `tryAsync` should be added.

The functions here basically accomplish the same thing as the ones in the starter-project. I've just been using these ones instead, and am more familiar with how the logic is set up, so chose to write them as so. Plus they look a bit cleaner IMO.

**Comment by ewilz** [comment]
Though `tryAsync` and another function `ensureDirectoryExistence` tend to be used often in deploy scripts as well. Haven't been writing deploy scripts for a while. Wonder if we need another helper module that's not specifically for tests eventually

File: src/helpers/expectNonZeroAddressProvider.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export default function expectNonZeroAddressProvider (web3) {
+  function isChecksumAddress (address) {
+    address = address.replace('0x', '')
+    var addressHash = web3.sha3(address.toLowerCase())
+    for (var i = 0; i < 40; i++) {
+      if (
```
Comment (ewilz) [inline src/helpers/expectNonZeroAddressProvider.js]: @mikec not sure I understand what's going on here? But when testing with legitimate addresses, I'm getting a failure.

**Comment by mikec** [comment]
think this is good to go! 👍

**Comment by mikec** [comment]
except for that `_latestTime` thing 😃

**Comment by ewilz** [comment]
wow @mikec , nice find

**Comment by ewilz** [comment]
FIXED.

**Pull request merged**